src/docs/implementation/ArcCommanderWiki/07_ARC-Synchronization.md

```diff
@@ -29,51 +29,59 @@ arc sync -r https://github.com/nfdi4plants/ARC/tree/Example-ARC.git
 **Note:** If your local repository is not connected to a remote repository, `arc sync` will only track LFS files and stage and commit changes to your local repository. The Commander will not pull or push any changes.
 
 
-## 6.1 Working on a second branch
+## 7.1 Working on a second branch
 
 *Problem:* 
 
-- No direct access rights to main branch
-
- or 
-- Many people work on repository, making direct pushing to main branch chaotic
+- No direct access rights to the main branch (which can be handled via the [DataHUB](../DataHub.md)) or 
+- Many people working on the same repository, making direct pushing to main branch chaotic.
 
 *Solution:* 
 
-Work on second branch, and contribute to the main branch using merge/pull requests
+Work on a second branch and contribute to the main branch using merge/pull requests
+
+*Workflow:*
+ 1. Use `arc get` to get the main branch of the repository.
+ 2. Work on the ARC.
+ 3. Use `arc sync -b SecondBranchName`.
+
+This will create a commit with your newest changes and push the commit to a new branch with the given name. When you finished editing your ARC, you can merge your progress into the main branch. 
 
-*Workflow*
- 1. use `arc get` to get main branch of repo
- 2. work on arc
- 3. use `arc sync -b secondbranchname`
+## 7.2 Git status
+
+You can check on which branch you are currently working on and the status of this branch by using
+
+```bash
+git status
+```
 
-    This will create a commit with your newest changes and push the commit to a new branch with the given name.
+This will print to the console, if you are behind, up to date, or ahead of the main branch.
 
-## 6.2 Setting a git user
+## 7.3 Setting a git user
 
 `git` always signs commits made with a `name` and `e-mail` adress of the user. These are then also used by `GitLab` and `GitHub` to assign the commits to user accounts. There are different ways to set this information.
 
-### Globally in git
+### 7.3.1 Globally in git
 
 The easiest way would be to do it directly globally using git:
 
 - `git config --global user.name "FIRST_NAME LAST_NAME"`
 
 - `git config --global user.email "MY_NAME@example.com"`
 
-### Locally for each ARC
+### 7.3.2 Locally for each ARC
 
-The problem with setting this information globally in git arises when you want to use different signatures for different repositories. E.g. when you develop software on `GitHub` and work on `ARCs` on the `nfdi4plants GitLab`.
+The problem with setting this information globally in git arises when you want to use different signatures for different repositories. E.g., when you develop software on `GitHub` and work on `ARCs` on the `nfdi4plants GitLab`.
```

Comment:
Rephrase to positively not call it a problem, but highlight the benefit: you **can** (this is optional) adapt the info for each individual ARC.